scripts/use-yarn-please.js

```diff
@@ -41,23 +41,6 @@ if (path.basename(npmPath) !== 'yarn.js') {
   process.exit(1);
 }
 
-checkNodeVersion();
-
-function checkNodeVersion() {
-  const nodeVersion = process.versions.node;
-  if (!nodeVersion) {
-    return;
-  }
-
-  const nodeMajorVersion = nodeVersion.split('.', 1);
-  if (nodeMajorVersion.length > 0 && parseInt(nodeMajorVersion[0]) >= 11) {
```

Comment:
I'd just update the version to 13, for anyone using the most current version. Thoughts?

Reply:
I agree with Tariq, checking against 13 seems like the best thing to do

Reply:
Is there any known issue with version > 13?